src/modules/client/components/ClientAlerts.tsx

```diff
@@ -0,0 +1,116 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import TitleCard from '@/components/elements/TitleCard';
+import theme from '@/config/theme';
+import {
+  clientNameAllParts,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertFieldsFragment,
+  ClientFieldsFragment,
+} from '@/types/gqlTypes';
+
+interface ClientAlertCardProps {
+  alert: ClientAlertFieldsFragment;
+  client: ClientFieldsFragment;
+  shouldShowClientName?: boolean;
+}
+
+const ClientAlertCard: React.FC<ClientAlertCardProps> = ({
+  alert,
+  client,
+  shouldShowClientName = false,
+}) => {
+  const priority = alert.severity || 'low';
+
+  const priorityColors: { [index: string]: any } = {
+    high: {
```

Comment:
Another question about colors - should this dictionary mapping alert priorities to their respective colors be moved into the theme file, instead of living here?